README.adoc

```diff
@@ -0,0 +1,25 @@
+= Jakarta Data
+:toc: auto
+
+== Introduction
+
+The Jakarta Data specification provides an API that allows easy access to database technologies. A
+Java developer can split the persistence from the model with several features,
+such as the ability to compose custom query methods on a `Repository` interface where the framework
+will implement it.
+
+== Meetings
+
+=== Calendar:
+* Europe: 
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=Europe/Athens++[Eastern],
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=Europe/Berlin++[Central],
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=Europe/Lisbon++[Western]
+
+* America: 
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Toronto++[Eastern],
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Chicago++[Central],
+link:++https://calendar.google.com/calendar/u/0/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc@group.calendar.google.com&ctz=America/Denver++[Western],
```

Comment:
I'm fine with the conversion to asciidoc, but switching the label for `Mountain` Time to `Western` Time will cause confusion.  I expect that most people in these regions would interpret "Western" to mean Pacific Time, not Mountain Time.  Can we leave this as it was?

Reply:
@njr-11 it might be a copy/paste mistake.
Sorry and thank you for seeing it.

Reply:
Thank you, @njr-11 
Update it.